spirv_msl.cpp

```diff
@@ -13319,12 +13319,17 @@ string CompilerMSL::builtin_qualifier(BuiltIn builtin)
 				SPIRV_CROSS_THROW("thread_index_in_simdgroup requires Metal 2.2 in fragment shaders.");
 			return "thread_index_in_simdgroup";
 		}
-		else
+		else if (execution.model == ExecutionModelKernel || execution.model == ExecutionModelGLCompute ||
+		         execution.model == ExecutionModelTessellationControl ||
+		         (execution.model == ExecutionModelVertex && msl_options.vertex_for_tessellation))
```

Comment:
What about fragment shaders?

Reply:
@cdavis5e Any comments here?

Reply:
Fragment shaders were handled by the previous `if`. There are no other places I can think of where the SIMD-group builtins are supported.